runtime/binding-mqtt-kafka/src/test/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSubscribeProxyIT.java

```diff
@@ -591,7 +613,21 @@ public void shouldExpireMessage() throws Exception
     {
         k3po.start();
         Thread.sleep(1500);
-        k3po.notifyBarrier("SEND_DATA");
+        k3po.notifyBarrier("WAIT_TIME_ELAPSED");
```

Comment:
If possible, let's avoid putting sleeps into the build as it doesn't take long for these to pile up and make the build a lot slower than necessary.

Perhaps we can take a different approach by initializing the message timestamp to be 1sec in the past, then the message is already expired and we don't need to wait?